### PR TITLE
changed main readme

### PR DESCRIPTION
**Pull request by pawelpluta**

### CODE REVIEW

**Review by bandrzejewski — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+
+One day, some developers started to write names of new programming methodologies that they identified. As we already followed
+Extreme Programming, we found out that they do not fall under this category. This is how Supreme Programming was born. By strong 
+suggestion of another team member - T.G. - we decided to bring them to a broader audience, so also other, struggling teams 
 can benefit from them. We care about developers, as we also are them, so we decided to describe those fascinating methodologies.
 
-> Writing means sharing. It's part of the human condition to want to share things - thoughts, ideas, opinions.
->
-> ~ Paulo Coelho
+We believe that you already might use them but do not know the proper approach. Review you knowledge, and bring those methodologies to live.
```
Comment (bandrzejewski) [inline README.md]: Not sure but: these* instead od those could be used. Also life* probably would be more accurate at the end.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
 # Supreme Programming Methodologies
 
-One day, some developers started to write names of new programming methodologies that they identified. By strong 
-suggestion of another team member - T.G. - we decided to bring them to broader audience, so also other, struggling teams 
+Extreme Programming Methodologies are nowadays used pretty common. That is very good, it brings joy to developers life,
```
Comment (bandrzejewski) [inline README.md]: It is very good, bringing joy to developer lives*

File: README.md [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
+as it allows to produce software of better quality. It allows for reducing costs within the company, as the amount of bugfixes that needs
+to be deployed is significantly lower than during development without Extreme Methodologies. Unfortunately, after some time
+of using them, you might ask yourself "Is there anything more I can do to make my software better?". Well, there is.
+
+One day, some developers started to write names of new programming methodologies that they identified. As we already followed
+Extreme Programming, we found out that they do not fall under this category. This is how Supreme Programming was born. By strong 
+suggestion of another team member - T.G. - we decided to bring them to a broader audience, so also other, struggling teams
```
Comment (bandrzejewski) [inline README.md]: suggestion of our team member*

File: README.md (hunk 1 above)
Comment (bandrzejewski) [inline README.md]: use them, but...*

File: README.md [hunk 4]
```diff
@@ -1,12 +1,16 @@
 # Supreme Programming Methodologies
 
-One day, some developers started to write names of new programming methodologies that they identified. By strong 
-suggestion of another team member - T.G. - we decided to bring them to broader audience, so also other, struggling teams 
+Extreme Programming Methodologies are nowadays used pretty common. That is very good, it brings joy to developers life,
+as it allows to produce software of better quality. It allows for reducing costs within the company, as the amount of bugfixes that needs
```
Comment (bandrzejewski) [inline README.md]: ...better quality, and reduces costs within the company, as...

File: README.md (hunk 1 above)
Comment (pawelpluta) [reply README.md]: Better

**Review by bandrzejewski — APPROVED** [review]

**Pull request merged**